### ISSUE TITLE
Accessibility: Issues on demoapp header buttons

### ISSUE THREAD

**Issue by EdwardCoyle** [infor-design/enterprise#2088]
**Describe the bug**
There is an issue on the Header's action button in many of our demoapp samples about there being no discernible text present.  This is detected while running the [aXe accessibility extension](https://chrome.google.com/webstore/detail/axe/lhdoppojpmngadmnindnejefpokejbdd?hl=en-US) for Chrome Dev Tools.

**To Reproduce**
[Originally documented here](https://github.com/infor-design/enterprise/pull/2050#issuecomment-488678556):

- Open any example page with Chrome.  Originally, we tested on:
  - http://localhost:4000/components/listview/example-index
  - http://localhost:4000/components/listview/example-mixed-selection
  - http://localhost:4000/components/listview/remove-clear
- Open dev tools to the aXe extension tab
- make browser window small (ie. phone size)
- refresh the page
- run aXe "ANALYZE" and see that there's a violation about buttons having discernible text:

![Screen Shot 2019-05-02 at 12 22 21 PM](https://user-images.githubusercontent.com/3249601/57090528-00df9f80-6cd5-11e9-9549-7eccee371dff.png)

**Expected behavior**
There should be no aXe violations.

**Version**
- originally discovered in `v4.18.x`, but it's possible that it was present further back

**Platform**
n/a

**Additional Context**
It may be that this is not just a demoapp bug, but a bug in the accessibility of action buttons.  If that's the case, we should add aXe e2e tests that cover the changes made to fix this bug.

**Comment by tmcconechy** [comment]
Checked into this issue again. Its no longer reporting errors in axe so closing but i did update the $listview-tertiary-color color. (related but not)
